optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/UseCasesAndExamples-chapter.adoc

```diff
@@ -244,34 +242,35 @@ These realistic competitions provide an objective comparison of Planner with com
 [[nQueensProblemDescription]]
 ==== Problem Description
 
-Place n queens on a n sized chessboard so no 2 queens can attack each other.
-The most common n queens puzzle is the 8 queens puzzle, with __n = 8__:
+Place _n_ queens on a _n_ sized chessboard so that no two queens can attack each other.
+The most common _n_ queens puzzle is the eight queens puzzle, with __n = 8__:
 
 image::UseCasesAndExamples/nQueensScreenshot.png[align="center"]
 
 Constraints:
 
 * Use a chessboard of _n_ columns and _n_ rows.
 * Place _n_ queens on the chessboard.
-* No 2 queens can attack each other. A queen can attack any other queen on the same horizontal, vertical or diagonal line.
+* No two queens can attack each other. A queen can attack any other queen on the same horizontal, vertical or diagonal line.
 
-This documentation heavily uses the 4 queens puzzle as the primary example.
+This documentation heavily uses the four queens puzzle as the primary example.
```

Comment:
Ack, I can do that @ge0ffrey. There are certain circumstances where leaving a numeral is correct (these are outlined in the IBM Style Guide), but generally speaking, and in the case of the Queens, it should be spelled out.